PIRMovementASM/PIRMovementASM/main.asm

```diff
@@ -0,0 +1,16 @@
+start:
+	ldi r24, 0b11111111 // Set value to register 24
```

Comment:
Comments providing info about the purpose of a line of code is of much more value than simply stating what the code does. In this example I would have liked to know the meaning of this bit pattern and what it's going to be used for...